Sources/Penny/Handlers/AuditLogHandler.swift

```diff
@@ -0,0 +1,51 @@
+import DiscordBM
+import Foundation
+import Logging
+import NIOCore
+import NIOFoundationCompat
+import Models
+
+struct AuditLogHandler {
+    let event: AuditLog.Entry
+    let context: HandlerContext
+    var discordService: DiscordService {
+        context.services.discordService
+    }
+    var logger = Logger(label: "AuditLogHandler")
+
+    init(
+        event: AuditLog.Entry,
+        context: HandlerContext
+    ) {
+        self.event = event
+        self.context = context
+        self.logger[metadataKey: "event"] = "\(event)"
+    }
+
+    func handle() async throws {
+        switch event.action {
+        case .memberBanAdd:
```

Comment:
Report bans. I know it's in the audit-log too, but this is more convenient to have a message about in the moderators channel.